static/app/views/dashboardsV2/manage/dashboardList.tsx

```diff
@@ -78,6 +123,30 @@ function DashboardList({organization, location, dashboards, pageLinks}: Props) {
               })}
             </WidgetGrid>
           )}
+          renderContextMenu={() => (
+            <ContextMenu>
+              <MenuItem
+                data-test-id="dashboard-delete"
+                key="delete-dashboard"
```

Comment:
do we need to define a key here?

Reply:
Keys help React to identify which items have been changed, added or removed. It is usually necessary within arrays of elements and I believe you will see a warning in your editor/console if you do not define it. You can read about it in detail on this page  https://reactjs.org/docs/lists-and-keys.html#keys